content/en/docs/instrumentation/js/resources.md

```diff
@@ -66,15 +66,18 @@ In the above example, the SDK detected the process and also added the
 Below you will find instructions to get resources detected automatically for
 you. However, you might run into the situation that no detector exists for the
 resource you need. In that case you can use the environment
-`OTEL_RESOURCE_ATTRIBUTES` to inject whatever you need. For example the
-following script adds [Service][], [Host][] and [OS][] resource attributes:
+`OTEL_RESOURCE_ATTRIBUTES` to inject whatever you need. You can use the
+environment variable `OTEL_SERVICE_NAME` to set value of the `service.name`
+resource attribute. For example the following script adds [Service][], [Host][]
+and [OS][] resource attributes:
 
 ```console
-$ env OTEL_RESOURCE_ATTRIBUTES="service.namespace=tutorial,service.version=1.0,service.instance.id=`uuidgen`,host.name=${HOSTNAME:},host.type=`uname -m`,os.name=`uname -s`,os.version=`uname -r`" \
+$ env OTEL_SERVICE_NAME="app.js" OTEL_RESOURCE_ATTRIBUTES="service.namespace=tutorial,service.version=1.0,service.instance.id=`uuidgen`,host.name=${HOSTNAME:},host.type=`uname -m`,os.name=`uname -s`,os.version=`uname -r`" \
```

Comment:
Not sure why this was there (maybe someone wanted to add a default if $HOSTNAME is not set?):thinking: 
```suggestion
$ env OTEL_SERVICE_NAME="app.js" OTEL_RESOURCE_ATTRIBUTES="service.namespace=tutorial,service.version=1.0,service.instance.id=`uuidgen`,host.name=${HOSTNAME},host.type=`uname -m`,os.name=`uname -s`,os.version=`uname -r`" \
```

Reply:
@pichlermarc Even I was not sure why that is present. Anyways, I fixed the example as per your suggestion.

Reply:
thanks :slightly_smiling_face: